server/src/modules/users/application/use-cases/create-user/create-user-controller.ts

```diff
@@ -13,8 +13,16 @@ export class CreateUserController extends ControllerWithDTO<CreateUserUseCase> {
   }
 
   buildDTO(req: express.Request): Result<CreateUserDTO, Array<ValidationError>> {
+    let params: any = req.params
+    if(!Object.keys(req.params).length){
```

Comment:
this is what you're actually testing for, right?

```suggestion
    if (Object.keys(req.params).length === 0) {
```